PackageDefinitions/workingtitle-gx-metapackage.xml

```diff
@@ -0,0 +1,54 @@
+<AssetPackage Name="workingtitle-gx" Version="0.2.0">
+	<ItemSettings>
+		<ContentType>CUSTOM</ContentType>
+		<Title>Working Title GX</Title>
+		<Manufacturer/>
+		<Creator>Working Title</Creator>
+	</ItemSettings>
+	<Flags>
+		<VisibleInStore>false</VisibleInStore>
+		<CanBeReferenced>false</CanBeReferenced>
+	</Flags>
+	<AssetGroups>
+		<AssetGroup Name="workingtitle-aircraft-bonanza-g36">
```

Comment:
The Bonanza and C172 were included in the G1000 mod because it needed to ship compatibility fixes to work with the avionics knob on those planes.    You can omit those here to contain the changes.